app/src/main/java/bitcoin/wallet/core/Interfaces.kt

```diff
@@ -49,6 +49,12 @@ interface ISettingsManager {
 
     fun setBaseCurrency(currency: Currency)
     fun getBaseCurrency(): Currency
+
+    fun setUnlockAttemptsNumber(attemptsNumber: Int)
```

Comment:
Name is a bit confusing, makes you think that it sets attempts user already did. However it is attempts left.